package.json

```diff
@@ -22,7 +22,6 @@
     "ace-builds": "^1.2.6",
     "animate.css": "^3.5.2",
     "bootstrap": "^3.3.7",
-    "bootstrap-datepicker": "^1.6.4",
```

Comment:
Did you remove the wrong one? The import for `bootstrap-datepicker` still exists in app.scss#L11

Reply:
I made this today :)